gabbi/case.py

```diff
@@ -487,21 +502,16 @@ def _run_test(self):
         self.url = base_url
         full_url = self._parse_url(base_url)
 
+        # Replace variables in headers with variable values. This includes both
+        # in the header key and the header value.
+        test['request_headers'] = self._replace_headers_template(
+            test['name'], test['request_headers'])
+        test['response_headers'] = self._replace_headers_template(
```

Comment:
I'll need to look at this more closely in the daylight, but there's a chance the response half of this should be in `HeadersResponseHandler`, but maybe not. I've got insufficient brain at the moment to decide.